internal/common/certmanager.go

```diff
@@ -52,10 +58,46 @@ func checkClusterIssuer(wcClient *client.Client, clusterIssuerName string) func(
 			Kind:    "ClusterIssuer",
 			Version: "v1",
 		})
-		err := wcClient.Get(context.Background(), cr.ObjectKey{
-			Name: clusterIssuerName,
-		}, u)
+		err := wcClient.Get(ctx, cr.ObjectKey{Name: clusterIssuerName}, u)
 		if err != nil {
+			if errors.IsNotFound(err) {
+				// Cluster Issuer was not found so we'll check the status of the Job that creates it
+				var nestedErr error
+				logger.Log("ClusterIssuer '%s' is not yet found", clusterIssuerName)
+
+				// Get status of cluster issuer post-install Job
+				clusterIssuerJob := &batchv1.Job{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "cert-manager-giantswarm-clusterissuer",
+						Namespace: "kube-system",
```

Comment:
I'm guessing this is hardcoded in the app?

Reply:
Yeah, at least based on the name we set.